### PR TITLE
Sent email to invite

### PR DESCRIPTION
**Pull request by Tome94**
invites email working

### CODE REVIEW

File: frontend/src/components/NewGroup.tsx [hunk 1]
```diff
@@ -132,7 +133,25 @@ function NewGroup(props: { onClose: () => void; session: Session | null }) {
         console.error("There was a problem with the fetch operation:", error);
       });
   }
-
+  const handleButtonClick = () => {
+    friends.forEach((friend) => {
```
Comment (Winxton) [inline frontend/src/components/NewGroup.tsx]: do this in the createGroup API endpoint, instead of from the client.

**Comment by Winxton** [comment]
@Tome94 how's this going?

**Comment by Tome94** [comment]
emails are up for creating a new group and disbanding a group. Started working a instead on a hard delete to a soft delete ( in supabase  members -> active is a boolean.

File: backend/src/index.ts [hunk 2]
```diff
@@ -100,12 +101,16 @@ app.get("/api/user", async (req, res) => {
...
 
   // Get all groups from the database
   const accessToken = req.headers.access_token;
   const user = await getUser(accessToken);
-  const groups = await getMemberGroups(user.email);
+
+  let groups;
+
+  // Retrieve groups based on the 'active' query parameter
+  groups = await getMemberGroups(user.email, active === "true");
```
Comment (Winxton) [inline backend/src/index.ts]: const groups = ...

**Review by Winxton — APPROVED** [review]
LGTM

**Pull request merged**